packages/broker-service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:22-slim
```

Comment:
Can we use our docker base image in [this](https://hub.docker.com/r/orochinetwork/node).
If not, use node version 20 for stability and we will upgrade to version 22 later on.

Reply:
Using node 20 is fine. However, this repo can't be built on alpine-based image, because there's a dependency need g++, openssl to build when running yarn install, so if we use alpine image, we will got this error
![Screenshot 2024-08-30 at 11 53 27 AM](https://github.com/user-attachments/assets/6fe32bf7-ebc7-45f9-ae66-c63284d1d02c)
This error is mentioned on https://hub.docker.com/_/node, at `node:<version>-alpine` 
However, I will try to optimize this by leveraging multi-stage build, but because it's monorepo, so writing multi-stage built while a package depends on another is a bit challenge.

Reply:
Let me try to update our base image.